### DISCUSSION TITLE
Source data from Apache Sling

### DISCUSSION THREAD

**Discussion by preethamponnappa** [gatsbyjs/gatsby#28308] [Help]
I want to know weather its possible to connect Gatsby with Apache Sling. My requirement is to retrieve Content Stored in Apache Sling and render it in Gatsby static website using Sling API. I could not find a source plugin in the Gatsby site for Sling CMS

**Comment by LekoArts** [comment]
Gatsby can connect to anything that has a REST/GraphQL API. If there's no source plugin yet, feel free to create one and publish it or if you don't want to publish something you can write the whole logic in your `gatsby-node.js` file of your project.

The steps outlined here are the same though: https://www.gatsbyjs.com/docs/creating-a-source-plugin/
